<?xml version="1.0" encoding="UTF-8"?><testrun name="TestSelectSort2" project="BT-Library-Java" tests="5" started="3" failures="2" errors="0" ignored="0">
  <testsuite name="bt.tests.core.algorithms.sorting.TestSelectSort2" incomplete="true">
    <testcase name="test_SelectSort2_012345678_Expect_012345678_True" classname="bt.tests.core.algorithms.sorting.TestSelectSort2" time="0.004">
      <failure>junit.framework.AssertionFailedError: expected:&lt;[ 0, 1, 2, 3, 4, 5, 6, 7, 8]&gt; but was:&lt;[ 8, 1, 3, 4, 7, 5, 6, 2, 0]&gt;&#13;
	at junit.framework.Assert.fail(Assert.java:57)&#13;
	at junit.framework.Assert.failNotEquals(Assert.java:329)&#13;
	at junit.framework.Assert.assertEquals(Assert.java:78)&#13;
	at junit.framework.Assert.assertEquals(Assert.java:86)&#13;
	at junit.framework.TestCase.assertEquals(TestCase.java:253)&#13;
	at bt.tests.core.algorithms.sorting.TestSelectSort2.test_SelectSort2_012345678_Expect_012345678_True(TestSelectSort2.java:154)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="test_SelectSort2_63124805_10_7_9_15_11_13_12_14_Expect_0123456789101112131415_True" classname="bt.tests.core.algorithms.sorting.TestSelectSort2" time="0.002">
      <failure>junit.framework.AssertionFailedError: expected:&lt;[ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18]&gt; but was:&lt;[ 18, 1, 2, 4, 8, 0, 5, 9, 7, 17, 10, 15, 11, 13, 12, 14, 6, 3, 16]&gt;&#13;
	at junit.framework.Assert.fail(Assert.java:57)&#13;
	at junit.framework.Assert.failNotEquals(Assert.java:329)&#13;
	at junit.framework.Assert.assertEquals(Assert.java:78)&#13;
	at junit.framework.Assert.assertEquals(Assert.java:86)&#13;
	at junit.framework.TestCase.assertEquals(TestCase.java:253)&#13;
	at bt.tests.core.algorithms.sorting.TestSelectSort2.test_SelectSort2_63124805_10_7_9_15_11_13_12_14_Expect_0123456789101112131415_True(TestSelectSort2.java:103)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="test_SelectSort2_63124805_Expect_01234568_True" classname="bt.tests.core.algorithms.sorting.TestSelectSort2" incomplete="true"/>
    <testcase name="test_SelectSort2_876543210_Expect_012345678_True" classname="bt.tests.core.algorithms.sorting.TestSelectSort2" incomplete="true"/>
    <testcase name="test_SelectSort2_empty_Expect_empty_True" classname="bt.tests.core.algorithms.sorting.TestSelectSort2" incomplete="true"/>
  </testsuite>
</testrun>
